aws/resource_aws_dynamodb_tag_gen_test.go

```diff
@@ -0,0 +1,77 @@
+// Code generated by internal/tagresource/generator/main.go; DO NOT EDIT.
+
+package aws
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"github.com/terraform-providers/terraform-provider-aws/aws/internal/keyvaluetags"
+	"github.com/terraform-providers/terraform-provider-aws/aws/internal/tagresource"
+)
+
+func testAccCheckDynamodbTagDestroy(s *terraform.State) error {
+	conn := testAccProvider.Meta().(*AWSClient).dynamodbconn
+
+	for _, rs := range s.RootModule().Resources {
+		if rs.Type != "aws_dynamodb_tag" {
+			continue
+		}
+
+		identifier, key, err := tagresource.GetResourceId(rs.Primary.ID)
+
+		if err != nil {
+			return err
+		}
+
+		exists, _, err := keyvaluetags.DynamodbGetTag(conn, identifier, key)
+
+		if isAWSErr(err, "ResourceNotFoundException", "") {
+			continue
+		}
+
+		if err != nil {
+			return err
+		}
+
+		if exists {
+			return fmt.Errorf("resource (%s) tag (%s) still exists", identifier, key)
```

Comment:
Maybe a nit: should this say what kind of resource, i.e. `DynamoDB resource (%s) ...`

Reply:
Updated in ec2ea08f20e64ebae81916040559ef08ab2d6e1a 👍